personalize.admin.inc

```diff
@@ -335,11 +338,22 @@ function personalize_agent_list($agents = NULL, $allow_edit = TRUE, $allow_alter
     $sorted_agents[$agent_status][] = $agent;
   }
 
-  $build = array();
+  $build = array(
+    'personalizations_list' => array(
+      '#type' => 'container',
+      '#attributes' => array(
+        'id' => 'personalize-personalizations-list',
+      ),
+      'navigation_bar' => array(),
+    ),
+  );
+  $navigation_bar = array();
   foreach ($sorted_agents as $status => $agents) {
     if (empty($agents)) {
       continue;
     }
+    $link = l('<h3>' . $status_map[$status] . '</h3>', '', array('fragment' => 'personalization-group-' . $status, 'external' => TRUE, 'html' => TRUE));
```

Comment:
I too didn't realize theme_links wraps by <ul>, otherwise it would have been perfect there. Modified [here](https://github.com/acquia/personalize_module/commit/179ea0ad5d0db869a012e1d462b7e9f35a1fb70a).